xml/admin_monitoring_alerting.xml

```diff
@@ -1360,8 +1356,8 @@ monitoring:
         <term><option>OSD_NO_SORTBITWISE</option></term>
         <listitem>
           <para>
-            No pre-Luminous v12.y.z OSDs are running but the sortbitwise flag
-            has not been set. The sortbitwise flag must be set before Luminous
+            No &productname; &prevproductnumber; v12.y.z OSDs are running but the <option>sortbitwise</option> flag
```

Comment:
I dont understand these 2 sentences at all @jan--f , can you please re-phrase it better? What is Ceph pre-Luminous v 12.y.z? Looking at https://docs.ceph.com/docs/master/releases/, Ceph versions 12.y.z are simply Ceph Luminous. Maybe this way:
```suggestion
            No pre-Luminous (with &ceph; older that version 12)  &osd;s are running but the <option>sortbitwise</option> flag
```

Reply:
I'll wait for Jan on this one (and the one below). I don't know why I didn't check this sentence with Kai...

Reply:
So the intention of this is that users will see the warning `OSD_NO_SORTBITWISE` under two conditions: 1) the `sortbitwise` option is not set and 2) all OSDs daemons have at least the version luminous. To get rid of it the users should set the `sortbitwise` option.

The wording is a little weird indeed. "No pre-Luminous v12.y.z OSDs are running" means all OSDs are at least version 12.0.0. So your suggested change works fine thought it has a typo: `s/older that version/older than version/`

So maybe 

```suggestion
            No pre-Luminous (with &ceph; older than v12.0.0)  &osd;s are running but the <option>sortbitwise</option> flag
```